src/io.rs

```diff
@@ -204,7 +204,7 @@ pub fn load(vm: &Thread) -> Result<()> {
     vm.add_bytecode("io_bind", io_bind_type, 3, io_bind);
```

Comment:
I believe this should work (untested)
```
// Stack
//  [f, m, ()]
let io_bind = vec![Call(1), // [f, m_ret] // Call m ()
                   PushInt(0), // [f, m_ret, ()] // Add a dummy argument ()
                   TailCall(2)]; // [f_ret] // Call f m_ret ()

```
IO actions are implement as simply having an extra argument (`IO a = () -> a` so invoking an action just needs to add an extra dummy argument and calling the function.

I can add some documentation to the bytecode though if you want, currently the documentation is just "what the vm does".